<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayClient</class>
 <widget class="QMainWindow" name="PlayClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1459</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1451</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/gameGround.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn10">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn11">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn12">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bttn13">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn14">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>530</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/GameBackground.jpg);
border-bottom-color: rgb(0, 0, 127);
border-left-color: rgb(0, 0, 127);
border-right-color: rgb(0, 0, 127);
border-top-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 127);
border:5px solid rgb(0, 0, 103);;
border-radius:10px;
color: rgb(0, 0, 103);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/gameGround.jpg);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>   IP :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/gameGround.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/gameGround.jpg);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/gameGround.jpg);
border-image: url(:/new/prefix1/gameGround.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>113</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/GameBackground.jpg);
border-image: url(:/new/prefix1/gameGround.jpg);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>  My Card</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/CardFrame.jpg);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/GameBackground.jpg);
border-image: url(:/new/prefix1/gameGround.jpg);
alternate-background-color: rgb(0, 0, 0);
color: rgb(11, 0, 73);
color: rgb(85, 0, 255);
border:5px solid blue;
border-radius:10px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>   I will win this number of rounds in this hand</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/GameBackground.jpg);
border-image: url(:/new/prefix1/gameGround.jpg);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>  My Rival Card</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/CardFrame.jpg);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>60</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/gameGround.jpg);
color: rgb(0, 0, 0);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/gameGround.jpg);
color: rgb(0, 0, 0);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border-image: url(:/new/prefix1/gameGround.jpg);</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>62</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/GameBackground.jpg);
border-image: url(:/new/prefix1/gameGround.jpg);
color: rgb(0, 0, 0);
border-radius:10px;</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
